MixItUp.Base/ViewModel/User/UserViewModel.cs

```diff
@@ -224,7 +164,7 @@ public string RolesDisplayString
         public string FollowAgeString { get { return (this.FollowDate != null) ? this.FollowDate.GetValueOrDefault().GetAge() : "Not Following"; } }
 
         [JsonIgnore]
-        public bool IsSubscriber { get { return this.MixerRoles.Contains(MixerRoleEnum.Subscriber) || this.MixerRoles.Contains(MixerRoleEnum.Streamer); } }
+        public bool IsSubscriber { get { return this.MixerRoles.Overlaps(new MixerRoleEnum[] { MixerRoleEnum.Subscriber, MixerRoleEnum.Streamer }); } }
```

Comment:
Oooh, Overlaps(), haven't seen that one before :)

Reply:
Yeah! That's one of the cool benefits of HashSet, it's designed to be used for Intersects, Overlaps, etc. I've never used a HashSet before. I had to read a bit to understand them better.

Reply:
Oh, HashSets are magical for distinct data. That O(1) insert & lookup.....so good 👍